docs/source/upcoming_release_notes/1189-flow-switch.rst

```diff
@@ -0,0 +1,32 @@
+1189 flow-switch
+#################
+
+API Breaks
+----------
+- N/A
+
+Features
+--------
+- pcdsdevices now has a `digital_signals.py` module for simple digital io.
+
+Device Updates
+--------------
+- added `J120K` to `SxrTestAbsorber`, `XPIM`, `IM2K0`, `PowerSlits`
+
+New Devices
+-----------
+- `PPMCoolSwitch` ppms with cooling switch not a meter.
+- `WavefrontSensorTargetCool` WaveFrontSensors with a cooling switch.
```

Comment:
```suggestion
- `WaveFrontSensorTargetCool` WaveFrontSensors with a cooling switch.
```
spelling fix to make the sphinx auto link work

Reply:
sphinxed